src/Tribe/Capabilities.php

```diff
@@ -15,24 +46,62 @@ class Tribe__Events__Capabilities {
 			'delete_private_posts',
 			'delete_published_posts',
 			'publish_posts',
-		),
-		'author' => array( // full permissions for content the user created
+		],
+		'author' => [ // full permissions for content the user created
```

Comment:
* :warning: Array double arrow not aligned correctly; expected 6 space(s) between "'author'" and double arrow, but found 1.
* :no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 399fea04e46e8290dee70b82ebb39174</sub>